packages/commercetools/theme/layouts/default.vue

```diff
@@ -5,21 +5,24 @@
     <BottomNavigation />
     <AppFooter />
     <CartSidebar />
+    <Login />
   </div>
 </template>
 
 <script>
 import AppHeader from '~/components/AppHeader.vue'
 import BottomNavigation from '~/components/BottomNavigation.vue'
 import AppFooter from '~/components/AppFooter.vue'
-import CartSidebar from '~/components/CartSidebar.vue'
+const CartSidebar = () => import('~/components/CartSidebar.vue')
```

Comment:
```suggestion
const CartSidebar = () => import(/* webpackChunkName: cartSidebar */  '~/components/CartSidebar.vue')
```
It's helpful for debugging purposes to name async chunks

Reply:
yeah looks like nuxt jut uses [id]